### PR TITLE
BaSD-MS-08: Profiles - Components

### PR DESCRIPTION
**Pull request by nicolasblasco**
Tasks:
- Create a list component with the retrieved profiles from the server and be able to delete a profile
- Create a form component with the ability to create a profile and update a profile
- Add other components like buttons, inputs, selects, etc
- Follow the screenflow style 
- Validate the data in the post and put request

### CODE REVIEW

**Review by GuidoGlielmi — DISMISSED** [review]

**Review by GuidoGlielmi — APPROVED** [review]
For the next feature, a hover state over the buttons could be added.

File: .eslintrc.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
               "endOfLine": "auto"
             }
           ],
-        "react/prop-types": 0
+        "react/prop-types": 0,
+        "no-unused-vars": "off"
```
Comment (estefrare) [inline .eslintrc.json:33]:
```suggestion
        "no-unused-vars": "warn"
```

File: src/Components/Profiles/Form/form.module.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  margin: 1.2em 0 0 1em;
+  color: #FFFFFF;
+}
+
+select, input {
+  margin: 0.8em 0;
+}
+
+
+
```
Comment (estefrare) [inline src/Components/Profiles/Form/form.module.css:54]: remove extra black lines

File: src/Components/Profiles/Form/form.module.css [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  align-self: center;
+  margin-bottom: 1em;
+}
+
+.title {
+  margin: 1.2em 0 0 1em;
+  color: #FFFFFF;
+}
+
+select, input {
```
Comment (estefrare) [inline src/Components/Profiles/Form/form.module.css:49]: do not use pure tags

File: src/Components/Profiles/Form/index.js [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+              value={profileValue}
+              onChange={onChangeProfileValue}
+              placeholder={'Write a new profile'}
+              required
+              pattern="[A-Za-z ]*"
+            />
+          </div>
+        </div>
+        <div className={styles.button}>
+          <Button type={'submit'} label={'Confirm'} onClick={onSubmit} />
```
Comment (estefrare) [inline src/Components/Profiles/Form/index.js:118]:
```suggestion
          <Button type={'submit'} label={'Confirm'} />
```

**Review by estefrare — CHANGES_REQUESTED** [review]

**Pull request merged**